### ISSUE TITLE
async.dart build error

### ISSUE THREAD

**Issue by ijd-github** [a14n/dart-google-maps#64]
Thanks for the quick response to my previous issue - I've updated to v3.3.1. Now I get the following error in build using Dart v2.0.0 :

[error] The argument type 'String' can't be assigned to the parameter type 'Symbol'. (package:google_maps/util/async.dart, line 45, col 13)

**Comment by ijd-github** [comment]
Apologies full message was :

[SEVERE] build_web_compilers|ddc on package:google_maps/google_maps.module:
Error compiling dartdevc module:google_maps|lib/google_maps.ddc.js

[error] The argument type 'String' can't be assigned to the parameter type 'Symbol'. (package:google_maps/util/async.dart, line 45, col 13)

**Comment by a14n** [comment]
It looks like a build cache issue. Could you try to remove your `.dart_tool` folder and retry.

**Comment by ijd-github** [comment]
Thanks again for the fast response. I've rebuilt after deleting the .dart_tool folder but still get the same error - I've repeated this several times using both **webdev serve** and **pub run build_runner serve** with the same results. My pubspec.yaml folder contains the following
environment:
  sdk: '>=2.0.0-dev.67.0 <3.0.0'

dependencies:
  intl: ^0.15.6
  angular: ^5.0.0
  angular_components: ^0.9.0
  angular_forms: ^2.0.0
  angular_router: ^2.0.0-alpha+19
  rxdart: ^0.18.1
  google_maps: ^3.3.1
  build_runner: ^0.10.0

dev_dependencies:
  angular_test: ^2.0.0
  build_test: ^0.10.3+1
  build_web_compilers: ^0.4.1
  mockito: ^3.0.0
  pageloader: ^3.0.0
  test: ^1.3.0
  sass_builder: ^2.1.1

**Comment by sestegra** [comment]
I have the same error when launching `webdev serve`

```
Error compiling dartdevc module:google_maps|lib/google_maps.ddc.js

[error] The argument type 'String' can't be assigned to the parameter type 'Symbol'. (package:google_maps/util/async.dart, line 45, col 13)
```

**Comment by sestegra** [comment]
@a14n See https://github.com/reyerstudio/google-maps-markerwithlabel and launch `example/basic.html`to get this error

**Comment by a14n** [comment]
Sorry I made a mistake in a previous commit by refactoring a file that was in my `.pub-cache` :-(

Fix is coming.

**Comment by a14n** [comment]
released in 3.3.2
